t/config.t

```diff
@@ -147,7 +147,28 @@ subtest 'Test configuration override from file' => sub {
         ],
         'referers parsed correctly'
     );
+};
+
+subtest 'Test trimming white space from both ends of openqa.ini value' => sub {
 
+    my $t_dir = tempdir;
+    local $ENV{OPENQA_CONFIG} = $t_dir;
+    my $app  = Mojolicious->new();
+    my @data = (
```

Comment:
I'd find multi-line strings here easier to read ie.

    my $data = `
        [global]
        appname =    openQA   # foo
    `;

And you could include comments as well.

Reply:
I will using the multi-line string, but comments are not supported. The code does support the value with "#" comments. Do we need to support it?

Reply:
I don't think we need to support in-line comments after the key=value part, only complete commented line, if this is what you mean. This would also correspond to what is usually supported in .ini-format AFAIK

Reply:
In this pr, I prefer not support in-line comments after key=value.